solidity/contracts/isms/CctpIsm.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+import {Message} from "../libs/Message.sol";
+import {TypeCasts} from "../libs/TypeCasts.sol";
+import {TokenMessage} from "../token/libs/TokenRouter.sol";
+import {CctpMessageV2} from "../libs/CctpMessageV2.sol";
+import {IMailbox} from "../interfaces/IMailbox.sol";
+import {AbstractCcipReadIsm} from "./ccip-read/AbstractCcipReadIsm.sol";
+import {ICcipReadIsm} from "../interfaces/isms/ICcipReadIsm.sol";
+import {TypedMemView} from "@memview-sol/contracts/TypedMemView.sol";
+import {IMessageTransmitter} from "../interfaces/cctp/IMessageTransmitter.sol";
+
+contract CctpIsm is AbstractCcipReadIsm {
```

Comment:
Wondering if this should just be part of the `ITokenBridge`? To make managing artifacts easier? I.e. the ISM of `TokenBridgeCctp` should always be this ism no?

Reply:
Yeah, would like me to deploy it within the constructor?